social_distribution/settings.py

```diff
@@ -91,8 +92,12 @@
 
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': BASE_DIR / 'db.sqlite3',
+        'ENGINE': 'django.db.backends.postgresql',
```

Comment:
setup an environment for your local if this is how your database is setup, but dont commit this, others might have different stuff